packages/eas-cli/src/build/android/build.ts

```diff
@@ -25,8 +27,17 @@ export async function prepareAndroidBuildAsync(
     platform: Platform.Android,
     easConfig,
   });
-
   const { buildProfile } = buildCtx;
+
+  if (
+    buildProfile.workflow === Workflow.Generic &&
+    !(await fs.pathExists(path.join(commandCtx.projectDir, 'android')))
+  ) {
+    throw new Error(
+      '"android" directory not found. If you want to build managed project update builds.android.PROFILE_NAME.workflow in "eas.json".'
```

Comment:
```suggestion
      '"android" directory not found. If you're trying to build a managed project, set builds.android.PROFILE_NAME.workflow in "eas.json" to "managed".'
```

Reply:
Is it possible to replace `PROFILE_NAME` with the actual name?

Reply:
we use PROFILE_NAME this way for schema already

Reply:
I'm not sure if I know what you mean.

`builds.android.${commandCtx.profile}.workflow` should do the trick.